welcome.md

```diff
@@ -78,21 +76,6 @@ Sandra Meneses, one of the participants who submitted a PR
 ::::
 :::::
 
-## The PyMC project
```

Comment:
I would keep this, and also add more content on the homepage as proposed here: https://github.com/pymc-devs/pymc.io/pull/14#issuecomment-1133750515. Not very opinionated on this being on the homepage, but I think all activities in addition to maintaining pymc should be showcased somewhere around here

Reply:
I think the purpose is to show that we're doing more than Just PyMC. But then the activities listed are all very much in support of PyMC. For me it's a bit as if Pandas would say "we also write the Pandas documentation". It's not really surprising and thus has low information density.

And most of these are already on the page elsewhere. Office hours are in the announcements, the example gallery is in the navbar, PyMCon when it's relevant would show up in the announcements.

I do think some of these links would be useful to have here, but I think a user cares less about what we do and rather how he or she can actually accomplish whatever they came to the site for. So what if we replaced this with a: "Get Started" with links to:
* Installation instructions
* Beginner User Guide
* Example Gallery
* Discourse

Reply:
I also added some of these pointers now to the "what sets us apart section" under a new "Community" item.